src/www/views/servers_view/server_connection_indicator/index.ts

```diff
@@ -24,9 +24,13 @@ export enum ServerConnectionState {
 
 const ANIMATION_DURATION_MS = 1750;
 const ANIMATION_DELAY_MS = 500;
-const CIRCLE_SIZES = [css`large`, css`medium`, css`small`];
 
-@customElement('server-connection-indicator')
+// These must be CSSResult values to be embedded in the
+// CSS below, but the standard css literal doesn't take
+// literals for security reasons, and wrapping them in the normal
```

Comment:
Did you mean "doesn't take non-literals"?

Reply:
I meant literal like a fixed single value, i.e. the string "large"

Reply:
Hopefully that edit is clearer